e2etest/bgp_test.go

```diff
@@ -74,7 +96,11 @@ var _ = ginkgo.Describe("BGP", func() {
 
 		hostport := net.JoinHostPort(ingressIP, port)
 		address := fmt.Sprintf("http://%s/", hostport)
-		_, err = runCommand(true, "wget", "-O-", "-q", address, "-T", "60")
+		useDocker := true
+		if skipDockerCmd {
+			useDocker = false
+		}
+		_, err = runCommand(useDocker, "wget", "-O-", "-q", address, "-T", "60")
```

Comment:
Is there a way to use the variable `skip-docker` directly to avoid the `if` clause above ? e.g.

`_, err = runCommand(!skipDocker, "wget", "-O-", "-q", address, "-T", "60")`

Reply:
Due to change the message, I need to keep `if skipDockerCmd`, so I can changed it no longer.